package/src/components/Chat/hooks/handleEventToSyncDB.ts

```diff
@@ -97,6 +97,48 @@ export const handleEventToSyncDB = async <
     }
   }
 
+  if (type === 'notification.mark_read') {
```

Comment:
Shouldn't this work the same as `message.read` ? That one does not provide `last_read_message_id` currently - is this intentional ?

If we don't have issues with this we should unify the way they work.

Reply:
Well this is a valid point, i can unify them using or and storing `last_read_message_id` locally for both events makes a lot of sense.

Reply:
Well yes, it makes sense to unify them and have the `last_read_message_id` stored for both as we receive it for both the events.